saturn-r/r-jupyter-kernel

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I'm ok with `postBuild` not having a file extension because that script has a special meaning from the `repo2docker` project, but I think that for clarity all other shell scripts should have a `.sh` extension. Could you change this file to `r-jupyter-kernel.sh`?

Reply:
So I finagled around with this, because for some odd reason starting up a container with the image sometimes gets fussy about the permissions on the `r-jupyter-kernel` file, so let me test this myself before I change it. It's probably fine, just want to make sure. (Although, is the likelihood that we will actually use this `r-jupyter-kernel` step in the official image build? I wasn't sure if that's a legit thing to have in the image or if you prefer to tell people to add the IRkernel piece in start scripts.)